3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,144 @@
+//First Task
+
+function getFullname(firstname, surname) {
+    return firstname + " " + surname;
+}
+
+let fullName1 = getFullname("Komal", "Ranjan");
+
+let fullName2 = getFullname("Pradeep", "Leuaa");
+
+console.log(fullName1); 
+
+console.log(fullName2); 
+
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lady " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+let fullname1 = getFullname("Komal", "Ranjan", true);
+
+let fullname2 = getFullname("Komal", "Ranjan", false);
+
+console.log(fullname1); 
+
+console.log(fullname2);
+
+function getFullname(firstname, surname, useFormalName, title) {
+    if (useFormalName && title) {
+        return title + " " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+let maleFullname1 = getFullname("Pradeep", "Leuaa", true, "Lord");
+
+let femaleFullname1 = getFullname("Komal", "Ranjan", true, "Lady");
+
+let maleFullname2 = getFullname("Shruti", "Leuaa", false);
+
+let femaleFullname2 = getFullname("Parth Sarthi", "Leuaa", false);
+
+console.log(maleFullname1);  
+
+console.log(femaleFullname1);
+
+console.log(maleFullname2); 
+
+console.log(femaleFullname2); 
+
+//Second Task
+
+function getEventWeekday(daysFromToday) {
+    let today = new Date();
+
+    let weekdays = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+
+    let eventDayIndex = (today.getDay() + daysFromToday) % 7;
+
```

Comment:
this is unnecessary because the modulus returns the remainder which will be between 0 and 6. So it will never  be smaller then 0